{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "40764d4c-b581-48fd-a063-34d5b3be0dc6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "inData", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1604655189629, "stop": 1604655189647, "uuid": "03471089-2454-4922-a5c9-d8515abecd0c", "historyId": "babdba5e5940d3ba90e375f5ee53eb4e", "testCaseId": "f56f10c6b6679a14e1ccd1e8f399d334", "fullName": "test_userManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_userManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "13656-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_userManage"}]}